packages/flutter_tools/lib/src/devfs.dart

```diff
@@ -18,6 +18,21 @@ import 'dart/package_map.dart';
 import 'globals.dart';
 import 'vmservice.dart';
 
+String getDillPath(
```

Comment:
this should be the one place where dill file names are assembled so that they always have a consistent naming pattern.